src/state/utils.js

```diff
@@ -13,6 +15,15 @@ export const createAsyncThunk = (name, action, onSuccessMessage, customErrorMess
       }
       return result
     } catch (error) {
+      // Handle auth error
+      if (error === 'UNAUTHENTICATED') {
```

Comment:
See notes above (L30).

Reply:
@paulschreiber Can you add a little more context for this? Do you think we should handle other codes? or just 401?

Reply:
If we get a 401, we want to bounce them to a login screen. If we get a 403, we should display an error.